cmd/draft/up.go

```diff
@@ -194,3 +212,29 @@ func defaultDraftEnvironment() string {
 	}
 	return env
 }
+
+func parseIgnores() (*ignore.Rules, error) {
+	// default ignored extensions
+	defaultsIR := strings.NewReader(strings.Join(defaultIgnores, "\n"))
```

Comment:
Yes! I was wondering which approach you would prefer, so I first did the more complex one. No worry, I just changed it to do this.